### ISSUE TITLE
Training Code

### ISSUE THREAD

**Issue by gwxxx** [TencentARC/TimeLens#3]
When will the training code be released?

**Comment by Vibhuvan** [comment]
Hey, I dont understand how the training data format should since you have not mentioned it properly.

You have written the train_scripts/run_sft_qwen3_8b.sh script in this manner

model_path="/path/to/Qwen3-VL-8B-Instruct"
datasets="gemini_refined_data"
model_id="qwen3-vl-8b"
min_tokens=64
total_tokens=14336
fps=2
fps_max_frames=""
seed=42

But, here you haven't mentioned how the training data gemini_refined_data should be and how the structure should be. Can you please clarify on this?
